scripts/sync-and-publish

```diff
@@ -4,6 +4,8 @@ set -xeuo pipefail
 release=$1
 opt=${2:-false}
 nightly=false
+origin="Mattermost"
```

Comment:
Is it possible to fetch `origin` and `label` from arguments and make them mandatory.

So that we can distinguish packages by origin. As Mattermost we may want to set origin from CI/CD pipeline to determine officially generated packages. And for anyone who builds and publish to their own repository, they may want to use different origin and label value.

Reply:
pretty good point yes